### DISCUSSION TITLE
Redirection HTTPS behind sub domain

### DISCUSSION THREAD

**Discussion by general03** [requarks/wiki#7070] [Error / Bug Report] [answered]
I installed wikijs inside docker (all right) and when I want to activate "redirection HTTP to HTTPS", wikijs redirect me to the IP local and not with my domain.
How can I force setup domain ?
Thanks

**Comment by NGPixel** [comment]
Set the proper hostname under Admin > General.

**Comment by general03** [comment]
I already set this parameter, I have the issue only when I want to force redirect. On HTTP it is works well

**Reply by general03** [reply]
What I need to configure outside Wikijs to link my domain with the domain configurated in Admin > General ?

**Comment by general03** [comment]
Do I need to install reverse proxy ?

**Comment by general03** [comment]
There are http server on this librairy but I can't see where I can update it

**Comment by general03** [comment]
Nobody used wikijs and their HTTPS redirection settings ?

**Reply by NGPixel** [reply, marked as answer]
This is all documented in the docs:
https://docs.requarks.io/install/config#https

Follow the instructions, make sure the correct hostname is set in Admin > General, then enable SSL redirection under Admin > SSL.

**Reply by general03** [reply]
Yes sure I already do this but when I activated this option wikijs redirect me on the IP server (behind this domain) and not with the right domain

**Comment by general03** [comment]
Hello, I get always the issue. Can you give me some tips, nobody has this issue ?
